website/docs/registry/providers/docs.mdx

```diff
@@ -9,6 +9,8 @@ description: >-
 
 The [Terraform Registry][terraform-registry] displays documentation for the providers it hosts. This page describes the expected format for provider documentation.
 
+-> Please note that provider documentation currently isn't supported in the private registry of [Terraform Cloud/Terraform Enterprise](https://developer.hashicorp.com/terraform/cloud-docs/registry/publish-providers#release-files)
```

Comment:
In the docs, we try to avoid stacking notes because it is less readable, so I'd recommend removing this note and adding the text above.

```suggestion
```